src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -254,10 +254,18 @@ class InterpolateLayerCPUTest : public testing::WithParamInterface<InterpolateLa
 
         function = makeNgraphFunction(ngPrc, params, interpolate, "InterpolateCPU");
 
+        // To be able to run CPU tests on all shapes real thresholds should be ignored
+        interpolate->get_rt_info().insert({"enforceAllSupportedLayouts", true});
+
         if (selectedType.empty()) {
             selectedType = getPrimitiveType();
         }
         selectedType = makeSelectedTypeStr(selectedType, ngPrc);
+
+        const auto selLength = selectedType.length();
+        if (selLength > 1 && selectedType[selLength - 2] == 'U' && selectedType[selLength - 1] == '8') {
```

Comment:
quite strange hardcode

Reply:
you check that it's `U8` and change the type after that without any reason.
Moreover, the comparation looks so strange (we compare the chars instead of element type). Why do you use such constractions?

Reply:
@iefode , the reason for this is to be consistent with the following workaround where exactly the same thing is done in existing code.
Could you please take a look at it?
https://github.com/openvinotoolkit/openvino/blob/c81fa39d3d40e305aa3783de2376ca3b3326533b/src/plugins/intel_cpu/src/node.cpp#L436
https://github.com/openvinotoolkit/openvino/blob/c81fa39d3d40e305aa3783de2376ca3b3326533b/src/plugins/intel_cpu/src/node.cpp#L442

Reply:
@iefode , did you have an opportunity to take a look at the original workaround?